scripts/config_test_vm.psm1

```diff
@@ -220,24 +220,16 @@ function Import-ResultsFromVM
         $VMSystemDrive = Invoke-Command -Session $VMSession -ScriptBlock {return $Env:SystemDrive}
 
         # Copy kernel crash dumps if any.
-        if (!(Test-Path ".\TestLogs\$VMName\KernelDumps\")) {
-            New-Item -ItemType Directory -Path ".\TestLogs\$VMName\KernelDumps"
-        }
-
         Invoke-Command -Session $VMSession -ScriptBlock {
             if (!(Test-Path "$Env:SystemDrive\KernelDumps")) {
                 New-Item -ItemType Directory -Path "$Env:SystemDrive\KernelDumps"
             }
             Move-Item $Env:WinDir\*.dmp $Env:SystemDrive\KernelDumps -ErrorAction Ignore
         }
-        Copy-Item -FromSession $VMSession "$VMSystemDrive\KernelDumps" -Destination ".\TestLogs\$VMName\KernelDumps" -Force -ErrorAction Ignore 2>&1 | Write-Log
+        Copy-Item -FromSession $VMSession "$VMSystemDrive\KernelDumps" -Destination ".\TestLogs\$VMName" -Recurse -Force -ErrorAction Ignore 2>&1 | Write-Log
```

Comment:
How is this related to changing the module_id? If it's not, can we move it to a different PR.